tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/prj.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_CPLUSPLUS=y
+CONFIG_NEWLIB_LIBC=y
```

Comment:
What are those lines doing? Are those Zephyr constants?

Reply:
Same as globing. These are features provided by Zephyr’s Kconfig configuration system. You can see [this](https://docs.zephyrproject.org/1.13.0/application/application.html#creating-an-application) (point 7). I guess they enable using C/C++ files.

Reply:
Okay, thanks!